Source/Core/Core/HW/WiimoteReal/IOWin.cpp

```diff
@@ -576,15 +577,45 @@ bool WiimoteWindows::IsConnected() const
   return m_dev_handle != nullptr;
 }
 
+// See http://wiibrew.org/wiki/Wiimote for the Report IDs and its sizes
+DWORD GetReportSize(u8 report_id)
```

Comment:
This should likely have `size_t` as its return type, since that's what `sizeof` returns. Calling code can then cast it if necessary.